.github/workflows/ci.yml

```diff
@@ -33,6 +33,34 @@ jobs:
     outputs:
       check_if_build: ${{ steps.check.outputs.check_if_build }}
 
+  build_lambda:
+    name: Build (Lambda)
+    if: (github.event_name == 'pull_request') || (github.event_name == 'push' && github.ref == 'refs/heads/main')
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./aws-lambda
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
```

Comment:
please replace it with `actions-rust-lang/setup-rust-toolchain@v1` similar to other jobs